lib/mix/lib/mix/tasks/xref.ex

```diff
@@ -294,31 +271,29 @@ defmodule Mix.Tasks.Xref do
   defp source_warnings(excludes, opts) do
```

Comment:
Nitpick: there is a comment above `## Warnings (unreachable + deprecated)` that I think should be `## Warnings (deprecated)`